src/sfizz/utility/StringViewHelpers.h

```diff
@@ -101,7 +101,12 @@ uint64_t hashNumber(Int i, uint64_t h = Fnv1aBasis)
 {
     static_assert(std::is_arithmetic<Int>::value,
                   "The hashed object must be of arithmetic type");
-    return hash(absl::string_view(reinterpret_cast<const char*>(&i), sizeof(i)), h);
+    union {
```

Comment:
Why was this needed? Some alignment issue?

Reply:
This block is not needed but this results in the better performance.
I changed this block because I found hashing process is costing a lot at first. but finally I found it is not the core issue.
The core issue is that the hash of ModKey region is not being calculated.

Reply:
I see thanks